changes/20057-connected-tweaks

```diff
@@ -0,0 +1 @@
+* Improved the accuracy of the heuristic used to deterimine if a host is connected to Fleet via MDM by using osquery data for hosts that didn't send a Checkout message.
```

Comment:
```suggestion
* Improved the accuracy of the heuristic used to determine if a host is connected to Fleet via MDM by using osquery data for hosts that didn't send a Checkout message.
```

Reply:
thank you! these go through chatgpt (😨) before they land the release notes so I'll leave it just so I don't lose your ✅ and have to bother you again + fight CI